### ISSUE TITLE
Eldritch spires warding doesn't dissipate

### ISSUE THREAD

**Issue by PootisManTF2** [TheCodex6824/ThaumicAugmentation#359]
Hello,

I've defeated many bosses at the top of several eldritch spires in the void dimension, and yet the warding stays active; I still can't break blocks nor can fly. I've seen that this issue was reported earlier this year and supposedly fixed, but it isn't the case apparently. 
I'm playing on a server running the 1.12.2 Pack, in case it matters.

              I made some changes in the latest version that should have fixed this - if they didn't, let me know and I'll look again.

_Originally posted by @TheCodex6824 in https://github.com/TheCodex6824/ThaumicAugmentation/issues/332#issuecomment-1475380635_

**Comment by TheCodex6824** [comment]
What is supposed to happen is that either when the boss dies or one tick after (depending if it died "normally" or not), the ward should go away. I'll have to do modpack testing for this, because I can't get it to happen for me with just a minimal set of mods. Did you do anything "weird" to kill the boss? This shouldn't happen anyway, but I wonder if there's some condition that causes things to not work.

**Comment by PootisManTF2** [comment]
No, I dont remember doing anything special to kill the bosses, I just used my sword...
